generated/nodejs/test/errorreporting.js

```diff
@@ -115,11 +105,36 @@ function createErrorEvents() {
       report_location: sourceLocation2
     }
   };
-  api.reportErrorEvent(formattedProjectName, event3, function(err, response) {
+
+  // report the three events in parallel
+  async.eachOf([event1, event2, event3], function(event, index, callback) {
+    api.reportErrorEvent(formattedProjectName, event, function(err, response) {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      console.log('Reporting error event %d' ,index + 1);
+      callback();
+    });
+  }, function(err) {
     if (err) {
       console.error(err);
```

Comment:
Similarly, you will have to invoke `createErrorEventsDoneCallback` with the error.